PresentScreenings/Menu Delegates/ProgramMenuDelegate.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using AppKit;
+
+namespace PresentScreenings.TableView
+{
+    /// <summary>
+    /// Program menu delegate, enables and disables the items of the program
+    /// menu, based on their tag value.
+    /// </summary>
+
+    public class ProgramMenuDelegate : NSMenuDelegate
+    {
+        #region Private Members
+        const int _plannerMenuItemTag = 601;
```

Comment:
To do this consequently for all comparable cases, it would become way out of scope.
Furthermore, I would prefer to set the tags in code instead of in the Interface Builder.